LocalWiki.php

```diff
@@ -432,6 +432,131 @@ function onBeforePageDisplay( OutputPage $outputPage ) {
+			'UseCustomBlockMessage' => false,
+			'Redirect' => null,
+			'RLModules' => [],
+			'RLStyleModules' => [],
+			'AdditionalFields' => [
+				'Text' => [
+					'label-message' => 'emailmessage',
+					'type' => 'textarea',
+					'required' => true
+					]
+				],
+			'FieldsMergeStrategy' => null
+		];
+
+		$wgContactConfig['banquyen'] = [
+			'RecipientEmail' => 'banquyen@lophocmatngu.wiki',
+			'SenderName' => 'Xử lý bản quyền WLHMN',
+			'RequireDetails' => true,
+			'NameReadonly' => false,
+			'EmailReadonly' => false,
+			'SubjectReadonly' => true,
+			'UseCustomBlockMessage' => false,
+			'Redirect' => null,
+			'RLModules' => [],
+			'RLStyleModules' => [],
+			'AdditionalFields' => [
+				'DiaChi' => [
+					'class' => 'HTMLTextField',
+					'label-message' => 'banquyen-label-diachi',
+					'help-message' => 'banquyen-help-giaithich2',
+					'required' => true
+				],
+				'ToChuc' => [
+					'class' => 'HTMLTextField',
+					'label-message' => 'banquyen-label-tochuc',
+					'help-message' => 'banquyen-help-giaithich3',
+					'required' => false
+				],
+				'ChucVu' => [
+					'class' => 'HTMLTextField',
+					'label-message' => 'banquyen-label-chucvu',
+					'help-message' => 'banquyen-help-giaithich4',
+					'required' => true
+				],
+				'SoDienThoai' => [
+					'class' => 'HTMLTextField',
+					'label-message' => 'banquyen-label-sdt',
+					'help-message' => 'banquyen-help-giaithich5',
+					'required' => true
+				],
+				'DoiTuong' => [
+					'class' => 'HTMLSelectField',
+					'label-message' => 'banquyen-label-luachon',
+					'options-message' => 'banquyen-list-luachon',
+					'help-message' => 'banquyen-help-giaithich7',
+					'type' => 'textarea',
+					'required' => true,
+				],	
+				'LienKet' => [
+					'label-message' => 'banquyen-label-url',
+					'help-message' => 'banquyen-help-giaithich8',
+					'type' => 'textarea',
+					'rows' => 5,
+					'required' => true
+				],
+				'NoiDung' => [
+					'label-message' => 'banquyen-label-giaithich',
+					'help-message' => 'banquyen-help-giaithich9',
+					'type' => 'textarea',
+					'rows' => 10,
+					'required' => true
+				],
+				'XacNhan1' => [
+					'class' => 'HTMLCheckField',
+					'label-message' => 'banquyen-label-xacnhan1',
+					'required' => true
+				],
+				'XacNhan2' => [
+					'class' => 'HTMLCheckField',
+					'label-message' => 'banquyen-label-xacnhan2',
+					'required' => true
+				],
+				'XacNhan3' => [
+					'class' => 'HTMLCheckField',
+					'label-message' => 'banquyen-label-xacnhan3',
+					'required' => true
+				],
+				'KySo' => [
+					'class' => 'HTMLTextField',
+					'label-message' => 'banquyen-label-chuky',
+					'help-message' => 'banquyen-help-giaithich6',
+					'required' => true
+				]
+			],
+			'FieldsMergeStrategy' => 'replace'
+		];
+			
+		$wgHooks['SkinAddFooterLinks'][] = function( Skin $skin, string $key, array &$footerlinks ) {
+			if ( $key === 'places' ) {
+				$footerlinks['lienhe'] = Html::element( 'a',
```

Comment:
Html is a deprecated class alias, add `use MediaWiki\Html\Html;` at the top with the other use statements.